src/layouts/components/Header.tsx

```diff
@@ -201,11 +238,11 @@ function DesktopNavbar(): JSX.Element {
   );
 }
 
-function TabletMenu({ toggleMenu }: { toggleMenu: () => void }): JSX.Element {
+function TabletMenu(props): JSX.Element {
```

Comment:
the previous way is easier to know what is inside the prop, since you're not declaring an interface for the prop, let's add your new prop using the existing way